kore/src/Kore/Step/Simplification/Not.hs

```diff
@@ -184,3 +209,14 @@ mkMultiAndPattern
     -> BranchT simplifier (Pattern variable)
 mkMultiAndPattern patterns =
     Foldable.foldrM And.makeEvaluate Pattern.top patterns
+
+{- | Conjoin and simplify a 'MultiAnd' of 'Predicate'.
+ -}
+mkMultiAndPredicate
+    :: (SimplifierVariable variable, MonadSimplify simplifier)
+    => MultiAnd (Predicate variable)
+    -> BranchT simplifier (Predicate variable)
+mkMultiAndPredicate predicates =
```

Comment:
Why not `MultiAnd.toPredicate`?

Reply:
That's `Kore.Predicate.Predicate.Predicate` and this is `Kore.Internal.Predicate.Predicate`. Perhaps the latter needs a better name.